R/pivot-long.R

```diff
@@ -65,6 +65,7 @@ pivot_longer <- function(df,
       names_pattern = names_pattern,
       col_ptypes = col_ptypes
     )
+    spec <- check_spec(spec)
   } else {
     spec <- check_spec(spec)
   }
```

Comment:
Oh actually, now that `check_spec()` is called in both branches, you could simplify the code but doing the checking after the if statement. 

And could you please make the same change in `pivot_wider()`?

Reply:
Yes, I was thinking to move it out as well, but then `else` branch doesn't do anything. But if you are OK, with that, I'll move it.
Will update `pivot_wider`.